rules/naming/src/ValueObject/PropertyRename.php

```diff
@@ -8,7 +8,7 @@
 use PhpParser\Node\Stmt\Property;
 use PHPStan\Type\Type;
 
-final class PropertyRename extends Property
+final class PropertyRename
```

Comment:
Ha, I haven't noticed this. What was it for?

Reply:
It was a surprise for me as well... I somehow by mistake extended Property...